pkg/clioptions/clusterdiscovery/cluster.go

```diff
@@ -56,6 +56,8 @@ type ClusterConfiguration struct {
 	// and dual-stack-specific tests are run
 	HasIPv4 bool
 	HasIPv6 bool
+	// IPFamily matches the kube e2e way
+	IPFamily string
```

Comment:
If anything, I'd rather describe this variable like described upstream in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/test_context.go#L173, e.g.

`IPFamily defines default IP stack of the cluster`